CMakeLists.txt

```
@@ -174,7 +176,6 @@ if (${ENABLE_UNIT_TESTS} OR ${ENABLE_INTEGRATION_TESTS} OR ${ENABLE_REGRESSION_T
       # Clone the submodule
       execute_process(COMMAND git submodule update --init --force -- external/Catch2 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
     endif()
```

Put the newline back.

```suggestion
    endif()

```